vuepress/docs/v7.0/docs/curate/ecr-uninstall-flow.md

```diff
@@ -1,24 +1,33 @@
-# Bundle uninstall
+# Bundle Upgrade, Downgrade, and Uninstall
+When an application bundle is installed in the Entando App Builder, or from the Entando Hub, the advantage is that you can upgrade, downgrade or uninstall at any time. You can upgrade the bundle as a whole, or when changes are made to a single component, you can upgrade just that component. 
 
-## Uninstall process
-To uninstall a bundle that was previously installed on an Entando App Engine, from Entando App builder UI go to 
-the **Component Repository** tab. For the bundle you want to uninstall, click on the `Installed` button next to it. This will open a pop-up window with the option to uninstall.
+## Upgrade or Downgrade Bundle Version
+From Entando App Builder's **Component Repository**, you can upgrade or downgrade the version of a bundle. 
 
 ![Uninstall flow](./img/uninstall-bundle.png)
 
-An initial check is made to verify none of the bundle components are in use. 
-If any of the components are in use, a popup will alert you. 
-Note that you can't force the removal of a bundle if its components are in use.
+1. Click the `Installed` button next to the bundle and a popup window opens with the options to update the bundle version or uninstall. 
```

Comment:
pop-up

"update or uninstall the bundle version" (reads better to have the verbs explicitly acting on the subject)